R/render.R

```diff
@@ -64,15 +64,10 @@ render <- function(input,
 
   # setup a cleanup function for intermediate files
   intermediates <- c()
-  on.exit(lapply(intermediates,
-                 function(f) {
-                   if (clean && file.exists(f))
-                     unlink(f, recursive = TRUE)
-                 }),
-          add = TRUE)
+  on.exit(if (clean) unlink(intermediates, recursive = TRUE), add = TRUE)
```

Comment:
@jjallaire In case you wonder the rationale behind the change here: there is no need to use a loop to remove the dirs: `unlink()` can take a vector of files/dirs, and it removes them only if they exists, so there is no need to check `file.exists()`, either.